lib/coursemology/evaluator/models/programming_evaluation.rb

```diff
@@ -30,7 +30,7 @@ def language=(language)
   def package
     @package ||= begin
       body = plain_request('courses/assessment/programming_evaluations/:id/package', id: id)
-      Package.new(StringIO.new(body))
+      Package.new(Coursemology::Evaluator::StringIO.new(body))
```

Comment:
Because I want a consistent API for Packages. Eventually I want to pass a stream in, see #10. Since I get an entire buffer at once now, the best is to use a StringIO.

Then there's the reason of why I need the custom StringIO -- that's because StringIO does not inherit from IO, rightly or wrongly.